core/src/main/clojure/xtdb/pgwire.clj

```diff
@@ -1587,6 +1656,7 @@
               (cmd-write-msg conn msg-command-complete {:command "COMMIT"}))
 
     :query (cmd-exec-query conn portal)
+    :prepare (cmd-prepare conn portal)
```

Comment:
are we missing a case for `:execute` here? or is that covered by handle-simple-query? (but handle simple query only deals with executing queries - not DMLs?

Reply:
heh, no, we don't need one now 🙂

so the bind for EXECUTE binds and returns the PREPAREd statement rather than the EXECUTE (`assoc inner` rather than `assoc stmt`), which means that rather than `statement-type :prepare`, it now carries the statement-type of the underlying statement - `:query` or `:dml`, the `:execute` never makes it that far.

saved me a whole bunch of duplicate code, that one 😊